src/moneypenny/config.py

```diff
@@ -40,3 +40,42 @@ class Configuration:
     Set by the ``DOCKER_SECRET_NAME`` environment variable,
     configured by the helm chart.
     """
+
+    config_dir: str = os.getenv(
+        "MONEYPENNY_CONFIG_DIR", "/opt/lsst/software/moneypenny/config"
+    )
+    """Directory where the Moneypenny ConfigMaps will be mounted.  Leave
+    at default in normal operation.
+    """
+
+    dossier_path: str = os.getenv("DOSSIER_FILE", "/opt/dossier")
+    """Path to the dossier file's mountpoint created in the initContainers.
+    Leave at default in normal operation.  Note that it does not contain the
+    filename (usually "dossier.json")
+    """
+
+    M: str = os.getenv("M_FILE", f"/{config_dir}/M/m.yaml")
+    """Path to M's standing orders.  Leave at default in normal operation.
+    """
+
+    quips: str = os.getenv("QUIP_FILE", f"{config_dir}/quips/quips.txt")
+    """Path to Moneypenny's quip file.  Leave at default in normal operation.
+    """
+
+    moneypenny_timeout: int = int(os.getenv("MONEYPENNY_TIMEOUT") or "300")
+    """Timeout (in seconds) to wait for all containers in the action pod to
+    complete.  Defaults to 300.
+    """
+
+    null_container_image: str = os.getenv(
+        "NULL_CONTAINER_IMAGE", "lsstsqre/farthing:latest"
```

Comment:
I'm confused by why you need this. My understanding is that the goal is to run a stack of containers one after the other. Why not just take the last container and run it as the main container and the rest as init containers? Seems equivalent and doesn't require this dummy container.

Reply:
removed